README.md

```diff
@@ -129,10 +129,34 @@ $ node src/app.js
 With the server running, open your favorite web browser and visit:
 http://localhost:3000
 
-If you want to specify the port number, set the `PORT` environment variable like so:
+### Environment Variables
+
+In order to mitigate uploading private keys, this repository uses a .env file for storing environment variables.
+
+For example, if you want to specify the port number, set the `PORT` environment variable like so:
+
+```.env
```

Comment:
This is not valid markdown. The ``` should be followed by a programming language name to give the proper syntax highlighting. Using `.env` will have no effect. In this case, I would use ```bash.

More info on syntax highlighting with Github flavored markdown: https://help.github.com/articles/creating-and-highlighting-code-blocks/

More info on bash: https://www.gnu.org/software/bash/manual/html_node/What-is-Bash_003f.html